lib/Makefile.am

```diff
@@ -22,6 +22,7 @@ include lib/str-repr/Makefile.am
 include lib/timeutils/Makefile.am
 include lib/secret-storage/Makefile.am
 include lib/logthrsource/Makefile.am
+include lib/http-auth/Makefile.am
 
```

Comment:
Why do we need to add this to lib?

Reply:
Without an interface how could one implement a pe-only plugin??